src/utils.py

```diff
@@ -194,3 +204,16 @@ async def get_repo(name: str) -> github3.repos.repo.Repository:
     github_token = await Secret.load("collection-registry-github-token")
     gh = await run_sync_in_worker_thread(github3.login, token=github_token.get())
     return gh.repository("PrefectHQ", name)
+
+
+def validate_view_content(view_dict: dict, variety: CollectionViewName) -> None:
```

Comment:
Should we also add additional checks to ensure there's no duplicates?

Reply:
I don't think there can be duplicates because we're using dictionaries to structure the data.

Reply:
duplicate keys should be impossible since `view_dict` is a dictionary, and the dict under each key should fail validation if there's a duplicate key inside